src/theme/components/template.ts

```diff
@@ -1,22 +1,25 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle } from "@chakra-ui/system";
-import { responsiveGap, responsivePadding } from "./global";
```

Comment:
These should be removed from the global file if not used elsewhere

Reply:
Done.